documentation/onboard-dpg-in-sdkautomation/.net/README.md

```diff
@@ -0,0 +1,21 @@
+# Add Autorest Configuration of .Net SDK
+
+## Parameter Description
+
+- `<ServiceName>`: The RP name, which is usually same as folder name in swagger.
+- `<NameSpace>`: Python package name.
+- `<Title>`: SDK Client Name
+
+## Autorest Configuration
+Please copy the following configuration into spec PR comment.
+~~~
+# azure-sdk-for-net-track2
+``` yaml
+title: <Title>
```

Comment:
Is your expectations that you will blindly take this content and add to the autorest.md file? Or do you plan to parse this information?

Reply:
`output-folder` and `require` is likely needed to be parsed and updated by all language script, as they are in relative path.

Reply:
yes. More details will be added to the document.